proof/refine/ARM/Interrupt_R.thy

```diff
@@ -356,8 +356,8 @@ lemma invoke_arch_irq_handler_corres:
 
 lemma invoke_irq_handler_corres:
   "irq_handler_inv_relation i i' \<Longrightarrow>
-   corres dc (einvs and irq_handler_inv_valid i)
-             (invs' and irq_handler_inv_valid' i')
+   corres dc (einvs and irq_handler_inv_valid i and simple_sched_action)
+             (invs' and irq_handler_inv_valid' i' and sch_act_simple)
```

Comment:
This is not about this PR, but we definitely can be more consistent in naming these concepts... 😆